WordPressKit/PostServiceRemoteREST.m

```diff
@@ -183,6 +183,33 @@ - (void)updatePost:(RemotePost *)post
            }];
 }
 
+- (void)savePost:(RemotePost *)post
```

Comment:
I might be overthinking this.
Do you think `savePost` could potentially be conflated with `updatePost`?  I mean, would it be reasonable to think that calling a save method should update the saved content in the canonical post record?  The reason I ask is I was looking at the behavior of the /autosave endpoint and got to thinking about expectations when I saw this:
```
// Drafts and auto-drafts are just overwritten by autosave for the same user if the post is not locked
// Non drafts or other users drafts are not overwritten. The autosave is stored in a special post revision for each user.
```
So depending on post status and authorship a revision might be created instead of updating the post. 

Would it make sense to name things `autosave` to reflect the endpoint's specific behavior?

Reply:
I believe you're right. I will update the name and add that comment in the method description here too.
As a side note, I would also like to add documentation to this endpoint after the main issue is merged.